### PR TITLE
fix: Update README.md with the correct status page links

### PR DESCRIPTION
**Pull request by adybose**
## Description

This PR fixes all the broken status page links in the **README.md** file.

## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [x] 🐛 Bug Fix
- [x] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Related Tickets & Documents

Fixes Issue #21
 
## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [x] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [ ] 🙅 no documentation needed

## [optional] Are there any post-deployment tasks we need to perform?
Once we have merged this PR, there should not be any more broken links to the status page in the repository's **README.md** file. However, there is one more place where this link needs to be updated, and that is the About section of the repository.

See Screenshot below:
Screenshot 2023-08-25 at 8 59 37 PM

## [optional] What gif best describes this PR or how it makes you feel?
giphy

### CODE REVIEW

**Review by takanome-dev — APPROVED** [review]
Sorry for the delay, LGTM :rocket:

**Review by jpmcb — APPROVED** [review]

**Comment by jpmcb** [comment]
Not sure why the Netlify actions fail. @brandonroberts / @bdougie any ideas? I'm also not familiar with this repo super well: seems that all the recurrent cron actions are failing: https://github.com/open-sauced/status/actions

We should look into that so the status site is up to date

**Comment by brandonroberts** [comment]
> Not sure why the Netlify actions fail. @brandonroberts / @bdougie any ideas? I'm also not familiar with this repo super well: seems that all the recurrent cron actions are failing: https://github.com/open-sauced/status/actions
> 
> We should look into that so the status site is up to date

The Netlify actions are failing because it expects a`package.json` to deploy this site but there isn't one here.

**Review by brandonroberts — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# [📈 Live Status](https://Deadreyo.github.io/upptime-test): <!--live status--> **🟩 All systems operational**
+# [📈 Live Status](https://status.opensauced.pizza/): <!--live status--> **🟩 All systems operational**
 
 This repository contains the open-source uptime monitor and status page for [NightKnight](https://www.linkedin.com/in/ahmed-mohamed-atwa/), powered by [Upptime](https://github.com/upptime/upptime).
```
Comment (bdougie) [inline README.md]:
```suggestion
This repository contains the open-source uptime monitor and status page for OpenSauced, powered by [Upptime](https://github.com/upptime/upptime).
```

Probably can remove NightNight while we are here.

**Comment by bdougie** [comment]
> The Netlify actions are failing because it expects a`package.json` to deploy this site but there isn't one here.

This was an oversight, I can fix. The status page was put together by our MLH fellow last year, and hasn't been touched in awhile.

**Comment by takanome-dev** [comment]
> seems that all the recurrent cron actions are failing

Yup, probably the uptime bot is using his `GH_PAT` that's why the actions are running successfully in [Deadreyo account](https://github.com/Deadreyo/status.opensauced) and all the statuses shown in the readme (this repo's readme) point to his actions

**Comment by bdougie** [comment]
Let's merge this, and focus on the other quick fixes separately

**Pull request merged**